### ISSUE TITLE
The symbol of Labtool 2.0 in the top bar should be a link to my page

### ISSUE THREAD

**Issue by yumoL** [UniversityOfHelsinkiCS/labtool#979]

**Comment by gotonode** [comment]
Resolved in #965.

**Comment by gotonode** [comment]
Done but not yet merged.

**Comment by hisahi** [comment]
Done in #1076
